lib/Dispatcher.js

```diff
@@ -39,29 +47,35 @@ module.exports = function () {
      * @throws {Error} if store is invalid
      * @throws {Error} if store is already registered
      */
-    Dispatcher.registerStore = function registerStore(store) {
-        if ('function' !== typeof store) {
```

Comment:
this function was wasting memory because everytime `Dispatcher()` was called it would create an other function with exactly the same code.